Backpack/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color name="bpk_dialog_background">@color/bpkWhite</color>
```

Comment:
Why not using `@color/bpkWhite` directly?

Reply:
I want to have a single source of truth for the dialog colour. It also allows to change dialog colour only without affecting other resources and change dialog colour by affecting bpkWhite colour, making this uni-directional dependency.